### PR TITLE
Add cast find block

### LINKED ISSUE #56: Document `cast find-block`
See https://github.com/gakonst/foundry/pull/766

### PR DESCRIPTION
**Pull request by Sabnock01**
Adds `cast find-block` to `cast.md`

Closes #56

### CODE REVIEW

File: src/reference/cast.md [hunk 1]
```diff
@@ -42,15 +42,16 @@ block-number             Prints latest block number
 call                     Perform a local call to <to> without publishing a transaction.
 calldata                 Pack a signature and an argument list into hexadecimal calldata.
 chain                    Prints symbolic name of current blockchain by checking genesis hash
-chain-id                 returns ethereum chain id
+chain-id                 Returns ethereum chain id
 code                     Prints the bytecode at <address>
-completions              generate shell completions script
+completions              Generate shell completions script
 estimate                 Estimate the gas cost of a transaction from <from> to <to> with <data>
+find-block               Prints the block number closes to the provided timestamp
```
Comment (mds1) [inline src/reference/cast.md:49]: Small typo 🙂

```suggestion
find-block               Prints the block number closest to the provided timestamp
```

File: src/reference/cast.md [hunk 2]
```diff
@@ -738,6 +739,27 @@ $ cast estimate --from 0xd8dA6BF26964aF9D7eEd9e03E53415D37aA96045 0xc18360217d8f
...
 
+#### `find-block`
+
+```ignore
+cast find-block --rpc-url <RPC_URL> <TIMESTAMP>
+```
+
+env: `ETH_RPC_URL`
+
+Prints the block number closes to the provided timestamp
```
Comment (mds1) [inline src/reference/cast.md:750]: Same typo

File: src/reference/cast.md (hunk 1 above)
Comment (Sabnock01) [reply src/reference/cast.md:49]: You'll never believe this but I saw that typo this morning in the `cast` cli and was going to open a PR to fix it but it seemed like an awfully small thing and here I am copying and pasting it here only minutes later.

Comment (mds1) [reply src/reference/cast.md]: Heh you’re right looks like I had the same typo in the original PR 😅

Comment (Sabnock01) [reply src/reference/cast.md]: I'll open a PR to fix it myself right now.

**Comment by onbjerg** [comment]
Thanks! :smile:

**Pull request merged**